script-nivel-2.js

```diff
@@ -0,0 +1,19 @@
+const list = [1, 3, 1, 5, 3, 2];
+displayData(formatResult(countList(list))) ;
+
+
+function countList (ls) {
+    const countMap = {}; // va contine nr de aparitii in lista
+    ls.forEach((x) => countMap[x] = (countMap[x] || 0) + 1);
+    return countMap;
+}
+
+function formatResult(obj) {
+    let result = "";
```

Comment:
In loc sa folosim forEach si sa avem o variabila drept container putem formula functia astfel
```js
function formatResult(obj){
  return Object.keys(obj).sort()
                         .map(key => `${key} - ${obj[key]} aparitii`)
                         .join('<br/>');
}
```
Scopul acestei modificari este sa urmam conventia de a folosi toate variabilele const, exceptand cazul in care nici o varianta a algoritmului nu ne permite